AsyncDisplayKit/ASTableView.mm

```diff
@@ -126,6 +126,16 @@ @interface ASTableView () <ASRangeControllerDelegate, ASDataControllerSource> {
 
 @implementation ASTableView
 
+void ASPerformBlockWithoutAnimation(BOOL withoutAnimation, void (^block)()) {
```

Comment:
Add a comment above this that describes the UIKit issue being worked around and/or references this PR.